Codecademy/gamut
# feat(Icons): add TikTokIcon

### Overview


Adds TikTokIcon


### PR Checklist

- [x] Related to JIRA ticket: [DOT-75]

#### Testing Instructions



1. Use the [Storybook preview](https://65eb584f6e8185d072799fa8--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon) to view the new icons
2. The TikTokIcon will be under "Vendor Icons". Ensure the icon look good in the preview.
<img width="150" alt="Screenshot 2024-03-08 at 10 29 32 AM" src="https://github.com/Codecademy/gamut/assets/29311886/86aa1c30-b5bc-4517-a731-79581195ee1f">

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/37311)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/5355)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-5355)   |




[DOT-75]: https://codecademy.atlassian.net/browse/DOT-75?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
